specification/security/resource-manager/readme.md

```diff
@@ -106,11 +106,31 @@ tag: package-composite-v3
 
 The following packages may be composed from multiple api-versions.
 
+
+### Tag: package-preview-2024-05
+
+These settings apply only when `--tag=package-preview-2024-05` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2024-05'
+input-file:
+  - Microsoft.Security/preview/2024-05-15-preview/securityConnectorsDevOps.json
+suppressions:
+  - code: LroLocationHeader
```

Comment:
Please include "where" in suppressions to know which paths are being suppressed. 
Else suppression is done for whole file.

Reply:
This suppression is needed for all of our long running operations so this would be a very long list, would it be fine to suppress the file?